client/layout/masterbar/woo.scss

```diff
@@ -1296,6 +1296,28 @@ $breakpoint-mobile: 660px;
 			}
 		}
 
+		.components-button {
+			--color-accent: var(--woo-purple-40);
+			--color-accent-60: var(--woo-purple-50);
```

Comment:
```suggestion
			--color-accent-60: var(--woo-purple-60);
```

The original hover state of the primary button is `--woo-purple-60`. Should we use --woo-purple-60 instead?

<img width="679" alt="Screenshot 2025-05-05 at 14 16 24" src="https://github.com/user-attachments/assets/986be72c-5dba-49a0-8cc9-fc18b2c8de06" />